pkg/patch/patch_test.go

```diff
@@ -13,7 +13,7 @@ func TestGeneratePatch_AppendSidecarFalse(t *testing.T) {
 		AppendInit:            false,
 		Init: InitMeta{
 			Ports:          "80,443",
-			ContainerImage: "111345817488.dkr.ecr.us-west-2.amazonaws.com/aws-appmesh-proxy-route-manager:latest",
+			ContainerImage: "111345817488.dkr.ecr.us-west-2.amazonaws.com/aws-appmesh-proxy-route-manager:v2",
```

Comment:
It's OK to leave the test out. It shouldn't matter which version of image it is in the test.